register/forms.py

```diff
@@ -23,22 +20,25 @@ def save(self, application):
 
     def update_forms(self):
         forms = self.fetch()
+        if not forms:
+            return []
         applications = map(self.map, forms)
-        return map(self.save, applications)
+        return [app.save() for app in applications]
 
 
 class TypeformFetcher(ApplicationFormFetcher):
     base_url = 'https://api.typeform.com/v1/form/'
 
-    def get_form_id(self):
-        return settings.REGISTER_APP['applications_form_id']
+    @property
```

Comment:
Do this and all the "Not implemented" functions need to be defined? If so, are they not being used?